library/patient.inc

```diff
@@ -1200,9 +1200,9 @@ function newInsuranceData(
         return false;
     }
 
-    // If empty dates were passed, then zero them.
+    // If empty dates were passed, then null.
     if (empty($effective_date)) {
-        $effective_date = '0000-00-00';
+        $effective_date = null;
     }
     if (empty($subscriber_DOB)) {
         $subscriber_DOB = '0000-00-00';
```

Comment:
@bradymiller Please check this also as this field is allowing `NULL` (at db level) but when you insert (modes activate) it will gonna throw error like `$effective_date`

Reply:
Will make this a separate issue since will involve some detective work (ie. search where this is used in codebase and ensure null (ie. blank return when query for it) won't break anything.